content/pt/application_programming_interface.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Interface de programação de aplicações
+status: Completed
+category: technology
+---
+
+## O que é
```

Comment:
I wonder if we have better ideas for the sections names (What it is, Problem it addresses, How it helps)

Reply:
Great point, I think it's better to discuss that with the team at the slack channel :D